packages/light-apps/package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "build": "yarn copy:wasm && SKIP_PREFLIGHT_CHECK=true craco build",
     "copy:wasm": "copyfiles -Eu5 \"../light/src/generated/kusama_cc3/kusama_cc3.wasm\" public",
-    "start": "yarn copy:wasm && BROWSER=none SKIP_PREFLIGHT_CHECK=true craco start",
+    "start": "yarn copy:wasm && TSC_WATCHFILE=UseFsEventsWithFallbackDynamicPolling BROWSER=none SKIP_PREFLIGHT_CHECK=true craco start",
```

Comment:
When running `yarn start`, I often have a very high CPU usage. Googled a bit, and found out a lot of people had the same issue. This guy offers a solution: https://blog.johnnyreilly.com/2019/05/typescript-and-high-cpu-usage-watch.html